src/Collection/LeeanRecent.php

```diff
@@ -0,0 +1,130 @@
+<?php namespace Leean\Widgets\Collection;
+
+use Leean\Widgets\Models\AbstractWidget;
+
+/**
+ * Class LeeanRecent.
+ *
+ * @package Leean\Widgets\Collection
+ */
+class LeeanRecent extends AbstractWidget
+{
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		parent::__construct( 'Leean Recent', 'Display a list of recent posts' );
+	}
+
+	/**
+	 * Add the menu select.
+	 *
+	 * @param array $instance
+	 */
+	public function form( $instance ) {
+		parent::form( $instance );
+
+		$post_types = get_post_types();
+
+		$selected = isset( $instance['post_type'] ) ? $instance['post_type'] : 'post';
+		$number = isset( $instance['number'] ) ? $instance['number'] : 3;
```

Comment:
Shouldn't we use `absint( $instance['number'] )` to get a number instead of a string.